frontend/src/types/SchedulingTypes.ts

```diff
@@ -15,6 +15,7 @@ export type Schedule = {
   recurringDonationId: string;
   recurringDonationEndDate: string;
   notes: string;
+  volunteerId?: number;
```

Comment:
also can update the `Scheduling/index.tsx` to remove volunteerIds and add `volunteerId: ""`